src/utilities/schemaHandlers.js

```diff
@@ -36,16 +70,9 @@ const preProcessHelper = (obj) => {
         prop.additionalProperties = { type: 'string' }
       }
 
-      // add default titles to dropdown of allowed types/ subschemas
       if (prop.anyOf) {
-        Object.values(prop.anyOf).forEach(option => {
-          // if the allowed type/ subschema is not a ref nor has a title,
-          // it defaults to <parentProp.title> option 1, <prop.title> option 2, ...
-          // we can convert to display the type name instead
-          if (!option.$ref && option.type && !option.title) {
-            option.title = option.type
-          }
-        })
+        const allowedTypes = ['string', 'number', 'boolean', 'integer']
```

Comment:
nit: I'd consider moving this to group them with the other consts in `processAnyOf`